charts/pega/README.md

```diff
@@ -1014,10 +1014,12 @@ Parameter   | Description   | Default value
 ---         | ---           | ---
 `hazelcast.image` | Reference the `platform/clustering-service` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_HAZELCAST_IMAGE:TAG`
 `hazelcast.clusteringServiceImage` | Reference the `platform/clustering-service` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_CLUSTERING_SERVICE_IMAGE:TAG`
-`hazelcast.migratorImage` | Reference the `platform/clustering-service-kubectl` Docker image that you downloaded and pushed to your Docker registry that your deployment can access. | `YOUR_MIGRATION_JOB_IMAGE:TAG`
 `hazelcast.enabled` |  Set as true if client-server deployment of Pega Platform is required, otherwise false. Note: Set this value as false for Pega platform versions below 8.6, if not set the installation will fail | `true`
 `hazelcast.clusteringServiceEnabled` |  Set as true if client-server deployment of Pega Platform is required, otherwise false. Note: Set this value as false for Pega platform versions below 8.8, if not set the installation will fail | `false`
-`hazelcast.migration` |  Set as true after creating prallel cluster to migrate the data and client switch over to new Hazelcast service | `false`
+`hazelcast.migration.enabled` |  Set as true after creating parallel cluster to migrate the data and client switch over to new Hazelcast service | `false`
```

Comment:
@taz-pega-work ,  this is not just for embedded to c/s hazelcast migration. It is to migrate from old hazelcast to new hazelcast without any downtime on running environment. It can be embedded to c/s or c/s to c/s hazelcast migration.
punctuation updated.